src/app/static/src/tests/components/plots/table/table.test.ts

```diff
@@ -125,7 +125,9 @@ describe('Table from testdata', () => {
         expect(wrapper.findAll('th').at(2).text()).toBe('Sex (Click to sort Ascending)');
         expect(wrapper.findAll('td').at(2).text()).toBe('Female');
         expect(wrapper.findAll('th').at(3).text()).toBe('HIV prevalence (Click to sort Ascending)');
-        expect(wrapper.findAll('td').at(3).text()).toBe('1.00%');
+        const div = wrapper.findAll('td').at(3).findAll("div")
```

Comment:
This was just a missing test? You didn't actually change the table behaviour.

Reply:
test needed to be updated because I added more data to the testdata, which the table uses as well.